### PR TITLE
Add variant_type parameter to allow the vcf_readcount_annotator to run snv and indels separately

### PR DESCRIPTION
**Pull request by susannasiebert**
By processing indels and snvs separately, we will be able to get around the issue of conflicting bam-readcount entries for snv and indels at the same position.

Previously, we would annotate variants that were not in the input bam-readcount files with all 0s. Therefore, we weren't able to run snvs and indels separately, because the two runs would've overwritten each other's results.

With this new option you can now run in `snv`-only mode which will ignore indels and not write 0s to them. `indel`-only mode will ignore snvs and not write 0s for them.

### CODE REVIEW

**Review by johnegarza — APPROVED** [review]
+1 looks good

**Comment by chrisamiller** [comment]
Does this eliminate the need to accept multiple input bam-readcount files? That's another potential source of conflicts between bam-readcount input (same variant in multiple files) that could be removed by restricting to one file.

+1

File: vcf_annotation_tools/vcf_readcount_annotator.py [hunk 1]
```diff
@@ -69,6 +75,22 @@ def parse_bam_readcount_file(args):
                     coverage[(chromosome,position,reference_base)] = parsed_brct
     return coverage
 
+def has_snv(entry):
+    for alt in entry.ALT:
+        alt = alt.serialize()
+        ref = entry.REF
+        if len(ref) == len(alt):
```
Comment (acoffman) [inline vcf_annotation_tools/vcf_readcount_annotator.py:80]: This is perfectly clear as is so perhaps not worth bothering with, but it looks like `has_snv` and `has_indel` differ only in a single line (the condition), they could probably be refactored. Though, its certainly possible the resulting indirection would be less clear than this version. Feel free to take or leave the suggestion :)

**Review by acoffman — APPROVED** [review]
LGTM! +1

**Comment by susannasiebert** [comment]
@chrisamiller good point. I pushed a commit to removed that feature. Since this change would break backwards compatibility we will need to make the new release as a new major version.

**Pull request merged**